features/handled_errors.feature

```diff
@@ -6,6 +6,7 @@ Scenario: Override errorClass and message from a notifyError() callback
     And the request is a valid for the error reporting API
     And the exception "errorClass" equals "Bar"
     And the exception "message" equals "Foo"
+    And the payload field "events.0.device.time" is not null
```

Comment:
I would like to verify the format of this field and also that it is the current time.